styleguide/source/assets/js/fitvids.js

```diff
@@ -69,7 +69,7 @@
           var videoID = 'fitvid' + Math.floor(Math.random()*999999);
           $this.attr('id', videoID);
         }
-        $this.wrap('<div class="fluid-width-video-wrapper"></div>').parent('.fluid-width-video-wrapper').css('padding-top', (aspectRatio * 100)+"%");
+        $this.wrap('<div class="fluid-width-video-wrapper"></div>').parent('.fluid-width-video-wrapper').css('padding-top', (56)+"%");
```

Comment:
Good call! Didn't realize fitvids.js was their actual plugin and that hub-cta-half-video.js is ours.